### DISCUSSION TITLE
How can I change the default theme's color?

### DISCUSSION THREAD

**Discussion by Rounak-stha** [vuepress/ecosystem#83] [Q&A] [answered]
I've added `.vuepress/styles/palette.styl`
And changed the `$accentColor` but I don't see any change.

**Comment by Mister-Hope** [comment]
This is VuePress v2, while what you said seems to belong to VuePress v1.

**Reply by Rounak-stha** [reply]
I am using V2. I can't seem to find any documentation for overriding the colors.
Would be helpful if you could link me to some.

**Reply by meteorlxy** [reply, marked as answer]
https://ecosystem.vuejs.press/themes/default/styles.html

**Reply by Rounak-stha** [reply]
The site's down! 😅
![image](https://github.com/vuepress/ecosystem/assets/66557682/a78ac4b2-dc8e-451f-8cb5-b9279bb4dfd2)

**Reply by Rounak-stha** [reply]
Its working now 
Thanks for the quick response. 🙏
